icons/gatsby/gatsby-original.svg

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="utf-8"?><svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 128 128"><style type="text/css">.st0{fill:#663399;}.st1{fill:#FFFFFF;}</style><g><circle class="st0" cx="14" cy="14" r="14"/><g><path class="st1" d="M6.2,21.8C4.1,19.7,3,16.9,3,14.2L13.9,25C11.1,24.9,8.3,23.9,6.2,21.8z"/><path class="st1" d="M16.4,24.7L3.3,11.6C4.4,6.7,8.8,3,14,3c3.7,0,6.9,1.8,8.9,4.5l-1.5,1.3C19.7,6.5,17,5,14,5c-3.9,0-7.2,2.5-8.5,6L17,22.5c2.9-1,5.1-3.5,5.8-6.5H18v-2h7C25,19.2,21.3,23.6,16.4,24.7z"/></g></g></svg>
```

Comment:
[CONTRIBUTING.md#organizational-guidelines](https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md#organizational-guidelines): .svg file contains one version of an icon in a "0 0 128 128" viewbox